data/man/vala-language-server.1.scd

```diff
@@ -0,0 +1,29 @@
+vala-language-server(1)
+
+# NAME
+
+vala-language-server - Code Intelligence for Vala
```

Comment:
and Genie!

Reply:
I guess the description on this GitHub repo should be adjusted too then, I took it from there

Reply:
updated